src/main/java/com/delphix/masking/initializer/maskingApi/SetupDriver.java

```diff
@@ -269,6 +279,31 @@ private void handleFileFormats(List<FileFormat> fileFormats) throws IOException
         }
     }
 
+    private void handleMountInformation(List<MountInformation> mountInformationList) {
+        for(MountInformation mountInformation: mountInformationList) {
+            PostMountInformation postMountInformation = new PostMountInformation(mountInformation);
+            apiCallDriver.makePostCall(postMountInformation);
+            Integer mountId = Integer.valueOf(postMountInformation.getId());
+            this.mountNameToId.put(mountInformation.getMountName(), mountId);
+            PutApiCall putConnectMountApiCall = new PutApiCall() {
+                @Override
+                protected String getEndpoint() {
+                    return String.format("mount-filesystem/%d/connect", mountId);
+                }
+
+                @Override
+                protected void setResponse(String responseBody) {
+                    this.setId(mountId.toString());
+                }
+            };
+            try {
+                apiCallDriver.makePutCall(putConnectMountApiCall);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Can we add an error message as well? "Mount created but unable to connect"?

Reply:
Done